android/src/test/kotlin/com/rudderstack/sdk/kotlin/android/plugins/devicemode/utils/MockCustomIntegrationPlugin.kt

```diff
@@ -0,0 +1,65 @@
+package com.rudderstack.sdk.kotlin.android.plugins.devicemode.utils
+
+import com.rudderstack.sdk.kotlin.android.plugins.devicemode.IntegrationPlugin
+import com.rudderstack.sdk.kotlin.core.internals.models.AliasEvent
+import com.rudderstack.sdk.kotlin.core.internals.models.GroupEvent
+import com.rudderstack.sdk.kotlin.core.internals.models.IdentifyEvent
+import com.rudderstack.sdk.kotlin.core.internals.models.ScreenEvent
+import com.rudderstack.sdk.kotlin.core.internals.models.TrackEvent
+import kotlinx.serialization.json.JsonObject
+
+class MockCustomIntegrationPlugin : IntegrationPlugin() {
```

Comment:
Let’s create a sample `CustomIntegration` plugin in the sample app and include clear documentation at the top of the class explaining how to use it.